### PR TITLE
CUDA BFloat16 gelu, hardswish, hardsigmoid

### PR DESCRIPTION
**Pull request by zasdfgbnm**

### CODE REVIEW

File: test/test_torch.py [hunk 1]
```diff
@@ -16897,29 +16897,30 @@ def test_exp_slow(self, device, dtype):
...
 
+    @precisionOverride({torch.bfloat16: 1e-2, torch.float: 0.0002, torch.double: 0.0002})
+    @dtypesIfCUDA(torch.float, torch.double, torch.bfloat16)
     @dtypes(torch.float, torch.double)
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
     def test_hardswish(self, device, dtype):
-        inputValues = [-1000, -4, -3, -2, 0, 2, 3, 4, 1000]
+        inputValues = [-1000, -4, -3, -2, 0, 2, 3, 4]
+        if dtype != torch.bfloat16:
+            inputValues.append(1000)
```
Comment (ngimel) [inline test/test_torch.py:16907]: what's up with this? Error too large?

Comment (zasdfgbnm) [reply test/test_torch.py:16907]: Yes

Comment (ngimel) [reply test/test_torch.py:16968]: Do you know if backward for hardswish and hardsigmoid is tested anywhere? Aha, looks like it is tested in test_nn, but only with gradcheck and only for float64. Ok, whatever.

**Comment by zasdfgbnm** [comment]
ping @ngimel

**Review by ngimel — APPROVED** [review]

**Pull request closed without merging**